guardian/prosemirror-typerighter
# feat: UI overhaul

## What does this change?

A UI overhaul, including
- more accessible matches
- headers in the sidebar to separate and explain match types

They're based on [this Figma file](https://www.figma.com/file/5RM0QzmQe8vCcVY9cX4KLL/Accessibility?node-id=619%3A11119), with one conscious alteration – I've darkened the orange match colour to `300` to allow the icon colour to remain white in all cases.

## How to test

These changes mostly concern the UI – there are no new unit tests. See success criteria.

Here's how it looks downstream in Composer, the Guardian's CMS:
 
<img width="1125" alt="Screenshot 2022-07-25 at 10 44 40" src="https://user-images.githubusercontent.com/7767575/180748366-45e548ae-1021-40ee-97fd-323de2e3d143.png">

To test yourself in Composer, follow the instructions introduced in [this PR.](https://github.com/guardian/prosemirror-typerighter/pull/213)

## How can we measure success?

Users find the new UI more accessible, the filter options are more clearly signposted, and it's clearer what each match colour means.
